assets/css/admin/orders/_refunds-modal.scss

```
@@ -0,0 +1,104 @@
+
+#edd-submit-refund-status {
+	text-align: center;
+	font-size: 1.2em;
+
+	.edd-submit-refund-message {
+		&:before{
+			font-family: dashicons;
+			font-size: 1.5em;
+			vertical-align: middle;
+			color: #fff;
+			border-radius: 16px;
+			margin: 5px;
+		}
+
+		&.success:before {
+			content: "\f147";
+			background-color: $wp-green-50;
+			padding-right: 1px;
+		}
+
+		&.fail {
+			display: block;
+			margin-bottom: 16px;
+
+			&::before {
+				content: "\f335";
+				background-color: $wp-red-50;
+			}
+		}
+	}
+}
+
+#edd-refund-submit-button-wrapper {
+	text-align: right;
+}
+
+.refunditems {
+	td,
+	th.check-column {
+		vertical-align: baseline;
+	}
+
+	.column-amount,
+	.column-subtotal,
+	.column-tax,
+	.column-discount,
+	.column-total {
+		width: 100px;
+	}
+
+	.edd-form-group__control {
+		display: flex;
+		align-items: center;
+
+		input {
+			background-color: transparent;
+			border: 0;
+			border-bottom: 1px solid;
+			border-radius: 0;
+			box-shadow: none;
+			text-align: right;
+			width: 100%;
+
+			&:disabled {
+				border-bottom: none;
+			}
+
+			&:focus {
+				border-bottom: 2px solid var(--wp-admin-theme-color-darker-10);
```

To avoid a jump by adding an additional pixel to the calculated height a `box-shadow` could be used in addition to the `1px border`.

```suggestion
				border-bottom: 1px solid var(--wp-admin-theme-color-darker-10);
				box-shadow: 0 1px 0 var(--wp-admin-theme-color-darker-10);
```

---

Holy moly, Spencer, you are a wizard indeed!